src/core/server/saved_objects/service/saved_objects_client.ts

```diff
@@ -334,7 +334,7 @@ export interface SavedObjectsResolveResponse<T = unknown> {
 /**
  * @public
  */
-export interface SavedObjectsOpenPointInTimeOptions extends SavedObjectsBaseOptions {
+export interface SavedObjectsOpenPointInTimeOptions {
```

Comment:
We no longer extend `SavedObjectsBaseOptions` to remove the `namespace` option, as this new `namespaces` option was added.

Reply:
See my comment in https://github.com/elastic/kibana/pull/109062#pullrequestreview-734416033, I think we should leave the `openPointInTimeForType` function as-is.

Reply:
I agree with your approach, so `typeToNamespacesMap` is no longer necessary here and I will remove it. However `SavedObjectsOpenPointInTimeOptions` had a `namespace?: string` option due to this extension, that needs to be changed to a `namespaces?: string[]`

Reply:
Oh, I was going too fast, yep agreed